src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  margin: 0 auto;
+  width: 140px;
+  height: 160px;
+  margin-top: 2px;
+}
```

Comment:
The styles for this container are optional, and may be what prevents the tests from coming through.